tests/Sdk/Unit/Core/Application/Lifecycle/Subscriber/InitializedEventSubscriberTest.php

```diff
@@ -48,6 +49,11 @@ class InitializedEventSubscriberTest extends Unit
      */
     protected CommandExecutorInterface $commandExecutor;
 
+    /**
+     * @var \SprykerSdk\Sdk\Core\Application\Service\ContextFactory
+     */
+    protected ContextFactory $contextFactory;
```

Comment:
I know, that I worked on it and didn't fixed properly, sorry. But, please add an interface for the factory and use an interface instead of class. The same for the other services that are using ContextFactory.

Reply:
fixed